### ISSUE TITLE
Improve support for transactions with value on send-tx

### ISSUE THREAD

**Issue by nventuro** [OpenZeppelin/openzeppelin-sdk#1214]
Currently, the only way to append value to a function call done via `send-tx` is with the `--value` flag. On interactive mode, we should detect if the selected function is `payable`, and if so prompt a value input.

Additionally, we should include the value on the 'calling function foo with arguments a, b, c' message.

**Comment by kamescg** [comment]
Did this ever get added?

**Comment by frangio** [comment]
@kamescg No, we haven't implemented this yet and it has very low priority for us.
